qa/tasks/mgr/dashboard/test_rbd.py

```diff
@@ -790,6 +790,7 @@ def test_move_image_to_trash(self):
 
     def test_list_trash(self):
         id = self.create_image_in_trash('rbd', 'test_rbd', 0)
+        self.assertStatus([200, 201])
```

Comment:
could we call this inside `create_image_in_trash`?

Reply:
Sure, just need to make `create_image_in_trash` an instance method.